CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - [UISAUTCOMP-111](https://issues.folio.org/browse/UISAUTCOMP-111) `useUsers` hook - add user ids to cache keys to fetch users when user ids change.
 - [UISAUTCOMP-112](https://issues.folio.org/browse/UISAUTCOMP-112) Extend auto-open a record functionality.
 - [UISAUTCOMP-113](https://issues.folio.org/browse/UISAUTCOMP-113) Re-position advanced search button when there is too little space in the search pane.
+- [UISAUTCOMP-117](https://issues.folio.org/browse/UISAUTCOMP-117) Provide deprecation notice to `useUserTenantPermissions.js`.
```

Comment:
`4.0.1` has been released already. Please create a new section for `4.1.0` release